sdk/route/src/automatic.ts

```diff
@@ -49,8 +49,8 @@ export class NttAutomaticRoute<N extends Network>
   extends routes.AutomaticRoute<N, Op, Vp, R>
   implements routes.StaticRouteMethods<typeof NttAutomaticRoute>
 {
-  override NATIVE_GAS_DROPOFF_SUPPORTED: boolean = true;
-  override IS_AUTOMATIC: boolean = true;
+  static NATIVE_GAS_DROPOFF_SUPPORTED: boolean = true;
+  static override IS_AUTOMATIC: boolean = true;
```

Comment:
```suggestion
```

you could just remove this. no need to override it; it's the default value.

Reply:
good idea, changed